.github/workflows/gradle.yml

```diff
@@ -51,6 +51,7 @@ jobs:
               - '.github/**'
               - 'buildSrc/**'
               - 'tools/**'
+              - '*.gradle'
```

Comment:
this is a change to make sure all builds build on gradle file changes.

Reply:
Got it, good to know. Thanks